### PR TITLE
homework 3 with methods, constructor, objects

### PR DESCRIPTION
**Pull request by scononen**

### CODE REVIEW

File: src/main/java/org/example/homework_nr_3/TemperatureConverter.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package org.example.homework_nr_3;
+
+public class TemperatureConverter {
+    double C;
+    double F;
+    TemperatureConverter (double C, double F) {
+        this.C = C;
+        this.F = F;
+    }
+        public static double toCelsius(double F) {
```
Comment (gh3rtzz) [inline src/main/java/org/example/homework_nr_3/TemperatureConverter.java]: remove the `double F` from the method. It'll work this way

File: src/main/java/org/example/homework_nr_3/TemperatureConverter.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                return (F-32)*5/9;
+        }
+        public static double toFahrenheit(double C){
```
Comment (gh3rtzz) [inline src/main/java/org/example/homework_nr_3/TemperatureConverter.java]: here also

File: src/main/java/org/example/homework_nr_3/Circle.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+
+public class Circle {
+
+    double radius;
+    Circle(double radius) {
+        this.radius = radius;
+    }
+
+    public static double calculateArea(double Pi, double radius){
```
Comment (gh3rtzz) [inline src/main/java/org/example/homework_nr_3/Circle.java]: remove the `double Pi, double radius` and for Pi use Math.Pi

File: src/main/java/org/example/homework_nr_3/Circle.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
+package org.example.homework_nr_3;
+
+
+public class Circle {
+    double radius;
+    Circle(double radius) {
+        this.radius = radius;
+    }
+
+    public static double calculateArea(double radius){
```
Comment (gh3rtzz) [inline src/main/java/org/example/homework_nr_3/Circle.java]: remove the static keyword

Comment (scononen) [reply src/main/java/org/example/homework_nr_3/Circle.java]: done

**Review by gh3rtzz — APPROVED** [review]
approved

**Pull request merged**